content/en/agent/guide/network.md

```diff
@@ -343,6 +343,16 @@ See [default NTP targets][2].
 {{% /tab %}}
 {{< /tabs >}}
 
+## Configure ports
+
+If you need to change an inbound port because the default port is already in use by an existing service on your network, use the `datadog.yaml` configuration file. You can find most of the ports in the **Advanced Configuration** section of the file:
+
+{{< agent-config type="advanced configuration" filename="datadog.yaml" collapsible="true" disable_copy="true">}}
```

Comment:
These are _really_ long embeds. Are they totally necessary?

Reply:
Yeah, I know :(. I wish there was a way to confine the embed to the lines you need. Maybe I should just use a code block instead.

Reply:
Okay, I replaced them with snippets :)